applications/CoSimulationApplication/tests/test_co_sim_io_py_exposure.py

```diff
@@ -3,6 +3,7 @@
 
 from KratosMultiphysics.CoSimulationApplication import CoSimIO
 from KratosMultiphysics import kratos_utilities as kratos_utils
+from KratosMultiphysics.testing.utilities import GetPython3Command
```

Comment:
why sticking only to `Python3`? May be we can generalize it to `GetPythonCommand` (thinking about future releases)?

Reply:
I intentionally made it python3, this is what I want, and not (assuming there will be) e.g. Python4, which most probably will behave differently (just like py2 and py3 did)